Update index.md (#1285)
SumoLogic/sumologic-documentation: docs/alerts/monitors/index.md

## Rules

> * To manage and/or view monitors, you'll need the **Manage** and **View Monitors** [role capabilities](/docs/manage/users-roles/roles/role-capabilities.md). [Learn more about permissions](/docs/alerts/monitors/edit-settings/#monitors-folder-permissions).

Reviewer: ```suggestion
* To manage and/or view monitors, you'll need the **Manage** and **View Monitors** [role capabilities](/docs/manage/users-roles/roles/role-capabilities.md). [Learn more](/docs/alerts/monitors/edit-settings/#monitors-folder-permissions) about controlling permissions at the monitor or folder level.
```

> * You need the **Manage** or **View Monitors** [role capability](/docs/manage/users-roles/roles/role-capabilities.md) to manage or view monitors, respectively. In addition, you can control permissions at a monitor or folder level. You will need Fine Grain permission feature enabled on your account. More information [here](/docs/alerts/monitors/edit-settings/#monitors-folder-permissions).

Reviewer: To confirm, is "Fine Grain" what it's called in the UI?

```suggestion
* You'll need the **Manage** or **View Monitors** [role capability](/docs/manage/users-roles/roles/role-capabilities.md) to manage or view monitors, respectively. To control permissions at a monitor level or folder level, you'll need the [**Fine Grain** permission feature More information](/docs/alerts/monitors/edit-settings/#monitors-folder-permissions) enabled on your account.
```
